binding/webgl_rendering_context.cc

```diff
@@ -347,15 +348,15 @@ napi_status WebGLRenderingContext::Register(napi_env env, napi_value exports) {
       NAPI_DEFINE_METHOD("uniform1f", Uniform1f),
       NAPI_DEFINE_METHOD("uniform1fv", Uniform1fv),
       NAPI_DEFINE_METHOD("uniform1i", Uniform1i),
+      NAPI_DEFINE_METHOD("uniform2f", Uniform2f),
 // uniform1iv(location: WebGLUniformLocation | null, v: Int32Array | ArrayLike<number>): void;
```

Comment:
Ditto with these - this is my rough  TODO list. Once I implement a method I delete the commented out declaration. Please do that for all others.

Reply:
I can delete, but "uniform2f" !== "uniform1iv"